docs/src/main/sphinx/installation/kubernetes.rst

```diff
@@ -0,0 +1,395 @@
+   :doc:`other clients </client>`. For this example,
+   :ref:`install the command-line interface <cli-installation>`, and connect to
+   Trino in a new console session.
+
+   .. code-block:: text
+
+       trino --server http://localhost:8080
+
+#. Using the sample data in the ``tpch`` catalog, type and execute a query on
+   the ``nation`` table using the ``tiny`` schema. Try other
+   commands to explore this data set:
+
+   .. code-block:: text
+
+       trino> select count(*) from tpch.tiny.nation;
+        _col0
+       -------
+         25
+       (1 row)
+
+       Query 20181105_001601_00002_e6r6y, FINISHED, 1 node
+       Splits: 21 total, 21 done (100.00%)
+       0:06 [25 rows, 0B] [4 rows/s, 0B/s]
+
+#. Once you are done with your exploration, enter the ``quit`` command in the
+   CLI.
+
+#. Kill the tunnel to the coordinator pod. The is only alive while the kubectl
+   process is running, so you can just kill the ``kubectl`` process that's
+   forwarding the port. In most cases that'll just mean pressing ``CTRL`` + ``C``
+   in the terminal where the port-forward command is running.
+
+Configuration
+-------------
+
+The helm chart uses the :doc:`Trino container image </installation/containers>`
+by default. The Docker image already contains a default configuration to get
+started, and some catalogs to allow you to explore Trino. Kubernetes allows you
+to mimic a :doc:`traditional deployment </installation/deployment>` by updating
+the ``values.yaml`` file. It's important to understand how files such as the
+Trino configuration, JVM, and various :doc:`catalog properties </connector>` are
+configured in Trino before updating the values.
+
+.. _creating-your-own-values-yaml:
+
+Creating your own values.yaml
+"""""""""""""""""""""""""""""
+
+When you use your own ``values.yaml``, you only override the values you specify.
+The remaining properties use their default values. Add a ``values.yaml`` with
+the following configuration:
+
+.. code-block:: yaml
+
+   image:
+     tag: "|trino_version|"
+   server:
+     workers: 2
+     config:
+       memory:
+         heapHeadroomPerNode: "1GB"
+       query:
+         maxMemory: "4GB"
+         maxMemoryPerNode: "1GB"
+   coordinator:
+     jvm:
+       maxHeapSize: "8G"
+   worker:
+     jvm:
+       maxHeapSize: "8G"
+
+These values are higher than the defaults and allow Trino to use more memory
+and run more demanding queries. If the values are too high, Kubernetes might
+not be able to schedule some Trino pods, depending on other applications
+deployed in this cluster and the size of the cluster nodes.
+
+#. ``.image.tag`` is set to the current version, |trino_version|. Set
+   this value if you need to use a specific version of Trino. The default is
+   ``latest``.
+#. ``.server.workers`` is set to ``2``. This value sets the number of
+   workers, in this case, a coordinator and two worker nodes are deployed.
+#. ``.server.config.memory.heapHeadroomPerNode`` is set to ``1GB``.
```

Comment:
I am inclined to suggest that we move the config stuff out into a separate section or remove for now .. the only thing that should really be needed is adjusting the JVM heapsize to the pod size .. for the rest the defaults should suffice .. and if not its an advanced topic to dive deeper into this and should not be in this simple beginner example